### PR TITLE
Pi server mvp

### PR DESCRIPTION
**Pull request by HyperbolicStudios**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+<<<<<<< Updated upstream
```
Comment (Ryast89) [inline .gitignore:2]: Same as before but this looks like it got changed by mistake to me

File: .vscode/launch.json
Comment (Ryast89) [inline .vscode/launch.json:1]: Was this file meant to be added?

File: database_init.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import database
+
+def init_clean_database():
+    #set urls, phase1, update_interval, update_toggle, update_now_requested for each game
+    for game in ['A', 'B', 'C']:
+        database.set_game_attr(game, "url", "https://hypixel.net/threads/hypixel-mini-mafia-iii-logical-fallacies-edition-game-over-town-wins.1857318/page-")
+        database.set_game_attr(game, "D1", 1)
+        database.set_game_attr(game, "update_interval", 3600)
```
Comment (Ryast89) [inline database_init.py]: Is this 1/hr? Could probably set this to 300, but not required right now

File: pi_app.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+#Scheduler job for checking if updates are desired. Pulled every 10 seconds.
+@scheduler.task('interval', id='do_job_A', seconds=10, misfire_grace_time=900)
+def job_A():
+    global update_time_log_A
+    global update_time_log_B
+    global update_time_log_C
+    for game in ['A', 'B', 'C']:
+        if database.get_game_attr(game, "update_toggle"):
+            #get current time since epoch
+            if int(time.time()) > update_time_log_A + database.get_game_attr(game, "update_interval"):
```
Comment (Ryast89) [inline pi_app.py:31]: Remove prints, otherwise looks great

File: .vscode/launch.json
Comment (HyperbolicStudios) [reply .vscode/launch.json:1]: Nope, deleted

File: database_init.py (hunk 2 above)
Comment (HyperbolicStudios) [reply database_init.py]: Changed

File: pi_app.py (hunk 3 above)
Comment (HyperbolicStudios) [reply pi_app.py:31]: Removed

**Review by Ryast89 — APPROVED** [review]

**Pull request merged**